### PR TITLE
OPS-4808 - Log secure

### PR DESCRIPTION
**Pull request by olivierlefloch**
Also merge https://github.com/Work4Labs/work4us/pull/10377

### CODE REVIEW

File: django_short_urls/views.py [hunk 1]
```diff
@@ -162,6 +162,10 @@ def new(request):  # pylint: disable=too-many-branches
                 status=HTTP_BAD_REQUEST,
                 message="%s may not contain a '/' character." % key)
 
+    statsd.increment(
+        'workforus.new',
+        tags=['prefix:' + unicode(params['prefix']), 'is_secure:' + unicode(request.is_secure())])
```
Comment (asiebert) [inline django_short_urls/views.py]: Is the prefix giving us any information regarding / is related to SSL in any way?
It's fine when exploring, but mixing two completely separate information increases the likelihood of tag combinations. Might want two separate metrics here.

Comment (olivierlefloch) [inline django_short_urls/views.py]: Well, we're expecting 100% of a single value for `is_secure` so I'm not worried about multiplying tag combinations in this case, and adding the prefix will allow us to monitor which pages create the most `new` short links so I think it's quite useful.

Comment (asiebert) [inline django_short_urls/views.py]: But the SSL information is irrelevant to "new" specifically − I feel it should be backward: why don't we monitor all requests to the shortener for their SSL status, tagging whether they are "get" or "API new" calls?

Comment (olivierlefloch) [inline django_short_urls/views.py]: Well I don't really care whether GET requests are done over SSL - we care for `new` because passwords are specified.

Comment (asiebert) [inline django_short_urls/views.py]: It's a nice-to-have information for stats on our visitors also − your call though, let's merge if that is skipped

Comment (olivierlefloch) [inline django_short_urls/views.py]: Let's keep this project simple - doesn't sound like we need this at this time.

**Pull request merged**